apps/s_server.c

```diff
@@ -966,6 +969,7 @@ const OPTIONS s_server_options[] = {
      "The number of TLSv1.3 session tickets that a server will automatically  issue" },
     {"anti_replay", OPT_ANTI_REPLAY, '-', "Switch on anti-replay protection (default)"},
     {"no_anti_replay", OPT_NO_ANTI_REPLAY, '-', "Switch off anti-replay protection"},
+    {"http_server_readmode", OPT_HTTP_SERVER_READMODE, 's', "file-opening mode for 'fopen' in -WWW and -HTTP"},
```

Comment:
I'd prefer a _binary mode_ option rather than setting the `fopen(3)` argument on the command line.

Reply:
Well, I was thinking of BS2000 where open-mode can be `r,tabexp=yes` org `r,type=record,forge=seq` but maybe this is over-engineering.

(Also later one might want to turn off ebcdic-ascii conversion for binary files, in such a case a flag will be better than a string.)